### DISCUSSION TITLE
Capstone ML Concrete analysis

### DISCUSSION THREAD

**Discussion by 19807311** [teamalgoritma/community#484] [Q&A] [answered]
Hi, Kak tanya ya
1. kalau mau bikin transformasi data dengan log atau sqrt, yang diberi log() atau sqrt()  itu targetnya atau prediktornya? atau semuanya? 
2. cara mengembalikan log itu dengan exp() ya? kalau sqrt() mengembalikan nilainya dengan apa ya? terus kalau misal prediktornya di log kan, hasil koefisiennya hrs kita ubah dulu kah sebelum digunakan untuk predict?
3. tanya juga di rubric finding right composition ini boleh dikasih hint dong kak? :)  terus ada pertanyaan "how many class" maksudnya kelas apa ya ?
![image](https://user-images.githubusercontent.com/114885734/207595510-7ccbc3f1-eeaa-433c-a53a-02b19c6881c7.png)

**Comment by amaliaasf** [comment, marked as answer]
Dear Bu Nana @19807311 

Terimakasih sudah menghubungi Mentor Algoritma.

1. Untuk transformasi data sebenarnya bisa dilakukan baik pada data prediktor ataupun target.
Catatannya: umumnya transformasi data (misalnya dengan log()), biasanya dilakukan pada variabel yg tidak berdistribusi normal. Jadi bisa dilihat persebaran data terlebih dahulu, untuk mengetahui variabel mana yang sebaiknya ditransformasi.

2. Benar sekali Bu Nana, untuk mengembalikan log( ) bisa menggunakan exp( ). Untuk mengembalikan sqrt( ), bisa menggunakan ^2 saja Bu.
​
3. Terkait rubrics Finding the Right Material Composition, Ibu diminta untuk memilih satu variabel prediktor yang ingin ditinjau lebih lanjut, kemudian Ibu membuat pengelompokkan. Misal Ibu mengambil `cement`, dan ingin membuat 2 kelompok (cement<250 dan cement>=250). Kemudian Ibu diminta mengecek apakah adanya perbedaan yang signifikan antara kedua kelompok tersebut dalam menentukan nilai target kita, yaitu strength pada beton. Untuk uji tersebut, Ibu bisa melakukan test Anova. Berikut referensi terkait uji ANOVA pada R: https://algotech.netlify.app/blog/one-way-anova-test-in-r/.

Demikian yang dapat kami sampaikan. Semoga membantu Bu :)

**Reply by 19807311** [reply]
ok thanks mbak
